debian/swtpm-tools.install

```diff
@@ -9,14 +9,14 @@ cat <<_EOF_
 /usr/bin/swtpm_setup
 /usr/share/man/man8/swtpm-create-tpmca.8*
 /usr/share/man/man8/swtpm-localca.8*
-/usr/share/man/man8/swtpm-localca.conf.8*
-/usr/share/man/man8/swtpm-localca.options.8*
+/usr/share/man/man5/swtpm-localca.conf.5*
+/usr/share/man/man5/swtpm-localca.options.5*
 /usr/share/man/man8/swtpm_bios.8*
 /usr/share/man/man8/swtpm_cert.8*
 /usr/share/man/man8/swtpm_ioctl.8*
 /usr/share/man/man8/swtpm_localca.8*
 /usr/share/man/man8/swtpm_setup.8*
-/usr/share/man/man8/swtpm_setup.conf.8*
```

Comment:
swtpm.spec.in would have to be changed as well.

Reply:
Hi @stefanberger, thank you for your review. I have changed it and pushed it again. If you have additional comments, please let me know.

Reply:
If you please do 
```
touch configure
make
```

it will create the new swtpm.spec that you can then also append to this PR.

Reply:
Please add the generated file to the patch and I'll merge.